### ISSUE TITLE
Text set to lumo-disabled-text-color by default

### ISSUE THREAD

**Issue by JaakkoHintsala** [stefanuebe/vaadin-fullcalendar#184]
Vaadin version 23.3.28
vaadin-fullcalendar version 6.2.0

text color is set to lumo-disabled-text-color by default
![kuva](https://github.com/stefanuebe/vaadin-fullcalendar/assets/87773094/ead7ad56-46fe-40eb-89e3-c7d75b828168)

I suppose it is caused by this
https://github.com/stefanuebe/vaadin-fullcalendar/issues/157#issuecomment-1357854234

I was able to fix this with some css:
`.fc.fix-text-color a:not(:any-link){
  color: var(--lumo-body-text-color);
}`
and then
`calendar.addClassName("fix-text-color");`

**Comment by stefanuebe** [comment]
Strange, this should be fixed already. I'll check that

**Comment by stefanuebe** [comment]
The full-calendar-styles.css contains this styles, that addresses the mentioned issue. 

```
.vaadin-full-calendar.fc {
    /*display: block;*/ /*not sure why this was here, keeping it for now, but breaks the normal displayment*/

    /* Fixes a color issue, where otherwise day numbers would appear as the disabled text color (Idk why)*/
    a:not(:any-link) {
        color: var(--lumo-body-text-color);
    }
    ...
}
```

And at least the demo with V24 shows the stylings correctly

<img width="228" alt="image" src="https://github.com/stefanuebe/vaadin-fullcalendar/assets/36725474/e5db685d-3aca-426c-9ecd-417ef68d021f">

So maybe V23 has a problem with nested css. I also had issues with nested css in a customer project, where they used older browser versions.

**Comment by stefanuebe** [comment]
Can you please elaborate, if the demo works for you or if it also shows grayed out day numbers? https://stefan.app.fi/fullcalendar-demo/

**Comment by JaakkoHintsala** [comment]
> So maybe V23 has a problem with nested css. I also had issues with nested css in a customer project, where they used older browser versions.

> Can you please check, if the demo works for you or if it also shows grayed out day numbers? https://stefan.app.fi/fullcalendar-demo/

The demo works with my browser (Firefox 115.7.0esr)
turns out the CSS exists but it is garbled out for some reason.
![kuva](https://github.com/stefanuebe/vaadin-fullcalendar/assets/87773094/037fe592-dd6c-4c32-b2ce-09d443059cc3)
The error says "Invalid property name"

**Comment by JaakkoHintsala** [comment]
It was related to the browser after all. When I opened the exact same calendar on chromium the CSS works.
![kuva](https://github.com/stefanuebe/vaadin-fullcalendar/assets/87773094/baaa8f52-36d6-4d87-b94d-cdaa8fc93cbc)

**Comment by stefanuebe** [comment]
Yeah, this "scrambled" css is also shown for me in Chrome. It works nevertheless, so no idea, why Chrome has issues showing nested css in its browser tools. 

But to allow usage of older browsers, I will revert the styling from nested to old school, full blown selectors with the next bugfix release.

**Comment by stefanuebe** [comment]
Release 6.2.1. Stylesheets should now be shown clean and work in older browser versions.

**Comment by JaakkoHintsala** [comment]
Thank you for the fix!
